### ISSUE TITLE
Build sample models in WordPress and Drupal? 

### ISSUE THREAD

**Issue by HobokenMartha** [GSA/Open-And-Structured-Content-Models#4]
Nice structures!  I wonder if there are plans to build sample pages using these structures in WordPress and Drupal...and maybe a third content management system?   Then test them with current federal content creators.

**Comment by NIAID1Thes** [comment]
Good idea! The feedback we've been getting is that people want to see real life examples, so one of our first site development priorities is to work on creating a repository for exactly this type of information.

**Comment by RoseTHERESA** [comment]
I would also like to see real-life examples and perhaps contribute one of my own (but not the first one). Does anyone have an example of an article they could just pop up here to get started?

**Comment by bonniestrong** [comment]
I'm going to post my 'real life example' in a new thread.  We are using Drupal, although in this case it's not particularly relevant as we will extend whatever content type we start with (Basic Page, Article, etc).
I can't upload xls or pdf files here as far as I can tell, so I've had to make images.  Not ideal.  But I'm not going to spend my time recreating in html either.  Life's too short :)
